data/json/recipes/armor/suit.json

```diff
@@ -3279,7 +3279,12 @@
       { "proficiency": "prof_articulation" },
       { "proficiency": "prof_quenching", "fail_multiplier": 2 }
     ],
-    "tools": [ [ [ "tongs", -1 ] ], [ [ "hotcut", -1 ] ], [ [ "metal_tank", -1 ] ], [ [ "water", -240 ] ] ]
+    "tools": [
+      [ [ "tongs", -1 ] ],
+      [ [ "hotcut", -1 ] ],
+      [ [ "metal_tank", -1 ] ],
+      [ [ "water", -240 ], [ "water_clean", -240 ], [ "water_clean", -240 ] ]
```

Comment:
Was it intentional to add `water_clean` twice here?

Reply:
oh whoops. I’ll fix this at home